dspace-server-webapp/src/test/java/org/dspace/app/rest/ConfigurationRestRepositoryIT.java

```diff
@@ -52,4 +56,15 @@ public void getAll() throws Exception {
         getClient().perform(get("/api/config/properties/"))
             .andExpect(status().isMethodNotAllowed());
     }
+
+    @Test
+    public void getExposed() throws Exception {
+        getClient().perform(get("/api/config/properties/search/exposed"))
+                .andExpect(jsonPath("$._embedded.propertyResources", hasSize(2)))
+                .andExpect(jsonPath("$._embedded.propertyResources", Matchers.containsInAnyOrder(
+                        ConfigurationMatcher.matchConfiguration("configuration.exposed.single.value"),
+                        ConfigurationMatcher.matchConfiguration("configuration.exposed.array.value")
+                )))
+                .andExpect(status().isOk());
+    }
```

Comment:
can we add a check also using the admin token? the goal is to have a test that would protect us against future refactoring. In future we could allow admins to retrieve additional properties but this search method must exposed just the public values according to the rest contract